exonum/src/helpers/fabric/mod.rs

```diff
@@ -349,14 +349,12 @@ pub trait CommandExtension {
 ///
 /// Services should provide implementation of this trait.
 pub trait ServiceFactory: 'static {
```

Comment:
It seems that naming became completely confused.

Reply:
Yes. Maybe some suggestions about how this could be named?
There is struct that creates service instances when new service is initialized and it is called (runtime::ServiceFactrory).
And there is struct that creates that ServiceFactory using arguments from config/command line (helpers::ServiceFactory).

Reply:
I'd suggest `helpers::ServiceFactory` => `helpers::ArtifactProvider` and `runtime::ServiceFactory` => `runtime::Deployable`.
@alekseysidorov  what do you think about such a naming?